workflow-rc1/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: workflow-rc1
+home: https://github.com/deis/workflow
+version: v2-beta
```

Comment:
yup, good catch; issue filed in deisrel to catch this next time: https://github.com/deis/deisrel/issues/66